resource_obmcs_identity_compartment.go

```diff
@@ -56,7 +56,7 @@ func updateCompartment(d *schema.ResourceData, m interface{}) (e error) {
 func deleteCompartment(d *schema.ResourceData, m interface{}) (e error) {
 	sync := &CompartmentResourceCrud{}
 	sync.D = d
-	return crud.DeleteResource(d, sync)
+	//return crud.DeleteResource(d, sync)
```

Comment:
it seemed to me the opposite was the desired case, the `fmt.Errorf` was the old version, and the `DeleteResource` was the intended current state. At least if I look at `updateCompartment` as an example.